WeakAurasOptions/AceGUI-Widgets/AceGUIWidget-WeakAurasTextureButton.lua

```diff
@@ -112,7 +112,7 @@ local function Constructor()
   texture:SetPoint("BOTTOMLEFT", button, 4, 4);
   texture:SetPoint("TOPRIGHT", button, -4, -4);
 
-  button:SetScript("OnEnter", function() Show_Tooltip(button, texture.name, texture.path:gsub("\\", "\n")) end);
+  button:SetScript("OnEnter", function() Show_Tooltip(button, texture.name, type(texture.path) == "string" and texture.path:gsub("\\", "\n") or texture.path) end);
```

Comment:
i added a string type check because previously i migrated texture.path to a number, but reverted it, and i still had some data with type == "number"
this can be safely reverted

Reply:
revert it!